README.md

```diff
@@ -10,6 +10,10 @@ Then, check out the repository and run the following inside it:
 
 `$ npm install`
 
+followed by:
+
+`$ npm install react-native-tileview; rm -rf react-native-tileview`
```

Comment:
So since this is an npm package, you install it then remove it from the current directory to avoid conflicts? That going to make git all weird because it will constantly say we deleted the directory. Maybe this whole thing should be a separate repo that we actually import from npm. Thoughts?